packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -1,19 +1,16 @@
 import { DirectionalHint } from '../../common/DirectionalHint';
-import { getScrollbarWidth, getRTL, Rectangle as FullRectangle, IRectangle } from '../../Utilities';
+import { getScrollbarWidth, getRTL, IRectangle } from '../../Utilities';
 import {
   IPositionDirectionalHintData,
   IPositionedData,
-  Point,
   ICalloutPositionedInfo,
   ICalloutBeakPositionedInfo,
   IPositionProps,
   ICalloutPositionProps,
   RectangleEdge,
+  IWindowWithSegments,
 } from './positioning.types';
-
-export class Rectangle extends FullRectangle {
```

Comment:
are you sure you want to just remove this ? we can leave this but not exporting it in `positioning/index.ts`.

Reply:
I talked with Jon. I think it's the right thing to do. The whole reasoning for this is literally the 2 casts below.